src/App.tsx

```diff
@@ -140,6 +141,9 @@ function App() {
                   {/* Links */}
                   <Col xs={0} sm={0} md={10} lg={10} xl={10} style={{ paddingRight: 10 }}>
                     <Row justify="end" gutter={15}>
+                      <Col>
+                        <WalletButtons></WalletButtons>
```

Comment:
It would be nice to style the Buttons in the header similar to the Support Button. Perhaps even move the wallet button to the very right, so that we have the buttons next to each other.

Reply:
good idea. will do!